integration/stacks_test.go

```diff
@@ -140,15 +143,123 @@ func deployStack(ctx context.Context, oktetoPath, stackPath string) error {
 	return nil
 }
 
-func destroyStack(ctx context.Context, oktetoPath, stackManifest string) error {
+func destroyStack(ctx context.Context, oktetoPath, stackManifest, dir string) error {
 	log.Printf("okteto stack destroy")
 	cmd := exec.Command(oktetoPath, "stack", "destroy", "-f", stackManifest)
 	cmd.Env = os.Environ()
-	cmd.Dir = stackGitFolder
+	cmd.Dir = dir
 	o, err := cmd.CombinedOutput()
 	if err != nil {
 		return fmt.Errorf("okteto stack destroy failed: %s - %s", string(o), err)
 	}
 	log.Printf("okteto stack destroy success")
 	return nil
 }
+
+func TestCompose(t *testing.T) {
+	if mode == "client" {
+		t.Skip("this test is not required for client-side translation")
+		return
+	}
+
+	ctx := context.Background()
+	oktetoPath, err := getOktetoPath(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	tName := fmt.Sprintf("TestStacks-%s", runtime.GOOS)
+	name := strings.ToLower(fmt.Sprintf("%s-%d", tName, time.Now().Unix()))
+	namespace := fmt.Sprintf("%s-%s", name, user)
+
+	t.Run(tName, func(t *testing.T) {
+		log.Printf("running %s \n", tName)
+		k8Client.Reset()
+	})
```

Comment:
since we are not running this test in two modes, you don't need to wrap the code in a t.Run(XXX  function.